### PR TITLE
[OPS-8013] New PHP and old-school block management.

### PR DESCRIPTION
**Pull request by cafuego**
This also includes updated user-agent and remote IP block config. It includes all *currently* active block rules from Reliefweb and HR.info, so should not cause undue problems. Also it means all rules are maintained even if the custom block  config from the stack is now mapped to the wrong filename. Yay.

I have built these as new images but not pushed them yet as the new stable tags.

### CODE REVIEW

File: alpine-php/php-k8s-v7/etc/nginx/nginx.conf [hunk 1]
```diff
@@ -139,8 +139,8 @@ http {
   ## Include the map to block HTTP methods.
   include /etc/nginx/map_block_http_methods.conf;
 
-  ## Include blacklist for bad bot and referer blocking.
-  include /etc/nginx/blacklist.conf;
+  ## Include blocklist for bad bot, referer and re mote IP blocking.
```
Comment (cafuego) [inline alpine-php/php-k8s-v7/etc/nginx/nginx.conf]:
```suggestion
  ## Include blocklist for bad bot, referer and remote IP blocking.
```

File: alpine-php/php-k8s-v8/etc/nginx/nginx.conf [hunk 2]
```diff
@@ -140,7 +140,7 @@ http {
   include /etc/nginx/map_block_http_methods.conf;
 
   ## Include blacklist for bad bot and referer blocking.
```
Comment (cafuego) [inline alpine-php/php-k8s-v8/etc/nginx/nginx.conf]:
```suggestion
  ## Include blocklist for bad bot, referer and remote IP blocking.
```

**Review by teodorescuserban — DISMISSED** [review]
lgtm

**Comment by cafuego** [comment]
Welp. So much for that idea! Back to the way we used to do things.

**Comment by teodorescuserban** [comment]
> Welp. So much for that idea! Back to the way we used to do things.

Basterds!

**Pull request merged**